modules/local/process/deseq2_qc.nf

```diff
@@ -40,13 +40,15 @@ process DESEQ2_QC {
     def software    = getSoftwareName(task.process)
     def label_lower = params.multiqc_label.toLowerCase()
     def label_upper = params.multiqc_label.toUpperCase()
+    def count_col = 2
+    if (task.process =~ /_SALMON/) {count_col = 3}

```

Comment:
We should move this logic to the main workflow. e.g. https://github.com/nf-core/rnaseq/blob/3643a94411b65f42bce5357c5015603099556ad9/rnaseq.nf#L130

Reply:
Thanks @lpantano !! Everything looks great 😎 Just need to make the changes as I suggested above.

Reply:
ok, I think I did it?...check it out